src/components/Image/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ImgHTMLAttributes, useState } from 'react';
+import { styled } from '@mui/system';
+import { Container } from '@mui/material';
+
+interface ImageDetails {
+  imageCustomise?: ImgHTMLAttributes<HTMLImageElement>;
```

Comment:
instead of passing an additional `imageCustomise` object, extend the props' interface